src/DotNetWorker.Grpc/protobuf/src/proto/FunctionRpc.proto

```diff
@@ -32,7 +32,8 @@ message StreamingMessage {
     WorkerInitRequest worker_init_request = 17;
```

Comment:
Make sure to remove proto changes in this PR once we have a new PR to bring in the new proto version